packages/mobile/src/screens/app-screen/AppTabScreen/useTabScreenOptions.tsx

```diff
@@ -5,31 +5,25 @@ import {
   StackNavigationOptions
 } from '@react-navigation/stack'
 import { markAllAsViewed } from 'audius-client/src/common/store/notifications/actions'
-import { Platform, Text } from 'react-native'
+import { Text } from 'react-native'
 import { useDispatch } from 'react-redux'
 
 import AudiusLogo from 'app/assets/images/audiusLogoHorizontal.svg'
+import IconCaretRight from 'app/assets/images/iconCaretRight.svg'
 import IconNotification from 'app/assets/images/iconNotification.svg'
 import IconSearch from 'app/assets/images/iconSearch.svg'
+import { IconButton } from 'app/components/core'
 import { useDispatchWeb } from 'app/hooks/useDispatchWeb'
 import { useNavigation } from 'app/hooks/useNavigation'
 import { open as openNotificationPanel } from 'app/store/notifications/actions'
 import { makeStyles } from 'app/styles'
 import { useThemeColors } from 'app/utils/theme'
 
-import { IconButton } from '../core'
+import { AppScreenParamList } from '../AppScreen'
 
-import { TopBarArrowBack } from './TopBarArrowBack'
-import { MainParamList, SearchParamList } from './types'
+import { AppTabScreenParamList } from './AppTabScreen'
 
 const useStyles = makeStyles(({ palette, spacing, typography }) => ({
-  root: {
-    height: Platform.OS === 'ios' ? 86 : 55,
```

Comment:
This wasn't needed anymore?

Reply:
with the safeview i think we are covered now. plus the 86 was quite tall on legacy iPhones